### PR TITLE
Get started - "top funnel" changes

### PR DESCRIPTION
**Pull request by omesser**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

## motivation and explanation

I'm issuing this PR as a start for some of the changes I had in mind.
I still believe our get started is complex, bloated and intimidating and this PR doesn't solve all of this. But wanted to keep the changes somewhat iterative to allow efficient discussion and and cut scope per change in case my ideas of a better get-started guide don't align with others.

## in this PR (HL changes)
- Get start index page got re-shuffled a bit. "Following This Guide" welcomes the reader and explains bout the scenarios/trails/tracks before the users "have to make choices" to try and guide them through this
- The "trails" are now consistently "data management" and "experiment management". Language was adjusted so "management" is a high-level term, and the individual chapters detail out what does it mean to "manage" data / experiments.
- some chapters reordered or unified or renamed. Now there are 3 chapters per trail;
  - `data and model access` - was moved away from get-started to `user-guide/data-management`, and renamed to avoid confusion with "data access" which has different meaning in SWE than what we refer to here.
  - `experiment versioning` is now ~live-tracking~ -> `experiment tracking`
  - `experiment management` is now ~experiment-collaboration~ -> `collaborating on experiments`
  - `experiment iterations` content was appended to `experimenting using pipelines`

## out of scope - in followup PRs
condensing, cutting some content / moving it to how-tos or use-cases section. Possibly cleaning up some distractions (tons of info/tip admons, side tracking the reader and links to other places harming a linear experience of going through a get-started tutorial.

- Re-thinking / focusing `experiment-pipeline` page to not mainly repeat `data-pipelines`
- Condensing the index page - only mentioning the first of each "trail" - try condensing the "trails" to single page

### open questions I still have in mind
- data: metrics-paramters-and-plots - should this be in experiments?
- pipelines - there's some ambiguity by "data pipelines" and "experiments" in DVC. actually experiments are persisted pipeline executions and I'm not sure it's clear enough. Also not sure it's easily digestable.

cc @daavoo @shcheklein and others for reviews and opinions.

Check out the review app here: https://dvc-org-get-started-top-nr5wyh.herokuapp.com/doc/start

### CODE REVIEW

**Comment by shcheklein** [comment]
@omesser there are broken links it looks like ^^ ?

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (shcheklein) [inline content/docs/start/data-management/metrics-parameters-plots.md:26]: this change it not needed I hope?

**Comment by omesser** [comment]
@shcheklein  - link issues fixed

File: content/docs/user-guide/how-to/discovering-and-accessing-data.md
Comment (dberenbaum) [reply content/docs/user-guide/how-to/discovering-and-accessing-data.md:2]: Another option would be to move it to the end of get started so it's there but less of a focus.

File: content/docs/start/index.md
Comment (dberenbaum) [inline content/docs/start/index.md:124]: I think @daavoo had put some of the exp mgmt stuff before pipelines for a couple reasons:
1. To make clear that pipelines weren't a prerequisite
2. Because exp mgmt/collaboration might be a more important value prop than the exp pipelines

If you are trying to keep these pages somewhat independent, WDYT about switching the order here?

**Review by dberenbaum — CHANGES_REQUESTED** [review]
Thanks @omesser! I like that it makes meaningful changes without doing so much that it's overwhelming.

Thoughts on the questions you listed:

>    data: metrics-paramters-and-plots - should this be in experiments?

It feels like we are caught in between introducing experiments at the end of the data management section and leaving them out. We should either add `dvc exp run` here or leave this page out completely.

>    pipelines - there's some ambiguity by "data pipelines" and "experiments" in DVC. actually experiments are persisted pipeline executions and I'm not sure it's clear enough. Also not sure it's easily digestable.

Yeah, there's duplication, which would be okay except it makes things unclear. It relates to https://github.com/iterative/dvc.org/pull/4375. I think we either need a separate pipelines trail or we need to fully explain what's the same and what's different between the data pipelines and experiment pipelines.

One idea to move this forward: 
1. Consolidate all the pipeline setup at the top of /doc/start/experiments/experiment-pipelines by referring back to /doc/start/data-management/data-pipelines and /doc/start/data-management/metrics-parameters-plots. We can show how to convert at most the first stage in example-get-started-experiments and mention that the rest follow that same data pipelines process.
2. Follow up after this PR with another attempt to add a separate pipelines trail or otherwise reorganize the pipelines content between the existing trails.

**Comment by shcheklein** [comment]
One comment re `One idea to move this forward:` and `trails`. The whole idea of trails was to make them self-sufficient (you don't have to go into another trail to see that pipeline or exps are a possible evolution of data - we kinda show you the next steps right there, logical to the project and the context you are in). 

That's why we have two separate self-sufficient e2e projects, for example.

If we feel that this is too complicated we should consider getting back to a single, simplified version of the get started. A single project, a few pages, etc, etc. The downside of that approach usually is that it's hard to start with both `dvc add` and `dvc exp run` and `dvc stage add`.

File: content/docs/user-guide/how-to/discovering-and-accessing-data.md
Comment (omesser) [reply content/docs/user-guide/how-to/discovering-and-accessing-data.md:2]: I want to take this opportunity to move content away from get-started, so moving it to use-guide/data-management as you suggested

File: content/docs/start/index.md
Comment (daavoo) [reply content/docs/start/index.md:124]: I think it might be a good time to make `Pipelines` a separate component

**Comment by daavoo** [comment]
> That's why we have two separate self-sufficient e2e projects, for example.
> If we feel that this is too complicated we should consider getting back to a single, simplified version of the get started. A single project, a few pages, etc, etc. The downside of that approach usually is that it's hard to start with both dvc add and dvc exp run and dvc stage add.

As we have been iterating, I am more strongly convinced that we should do isolated `Data`, `Experiments` and `Pipelines` as get-started pages. 

Made them about showing specific commands:

- `Data`: `add/push-pull/remote add`.
- `Experiments`: `DVCLive/exp show/exp branch/exp push-pull`.
- `Pipelines`: `stage add/exp run`.

On top of the get-started pages and as an optimal follow-up: A single, end-to-end, reproducible tutorial that builds on the 3 components. 
The tutorial doesn't explain any command, assumes get started has been followed, and focuses instead on explaining workflow around the components.

For anything outside that or more details, dispatch to User Guide.

File: content/docs/start/index.md
Comment (daavoo) [inline content/docs/start/index.md:81]: WDYT about not mentioning the subpages at this point and just directing the users to the first page of each "trail"?

I think it would :
- reduce verbosity
- save users from having to choose where to start
- save potential confusion/discussion about the subtitles

File: content/docs/start/experiments/experiment-pipelines.md
Comment (omesser) [reply content/docs/start/experiments/experiment-pipelines.md:213]: added similar paragraph to the beginning, and i'll compact and 🔪 this content more in a followup PR

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md:68]: I took another stab at it by modifying your suggestion a bit

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md:124]: @dberenbaum  - reordered

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md:81]: That's an interesting idea, but I think It would behave more consistently only if we actually unify data/exp to 1 page each (yes, it would be long and require scrolling.... but that's fine, who said a lot of short pages is better)
Ok if I try this in an individual followup PR and we'll give it a try and get a feel for it?

File: content/docs/start/index.md
Comment (dberenbaum) [reply content/docs/start/index.md]: As a counterpoint, DVC pipelines was the primary value prop for me adopting it as a user. It's not a replacement for Airflow, but that's not what I wanted during development where everything changed frequently. As you mentioned above @omesser, it's more like a build tool for ML. Framed that way, I think is has a lot of value.

**Comment by shcheklein** [comment]
@daavoo what you are describing - we had it all before in different ways and forms. Some considerations / experience:

- If you have a quick start page that is not reproducible - be prepared for a lot of questions "I'm trying to run it and it fails"
- If we have tree pages Data, Exp, Pipelines - the first one will be the main one (unless you do some explicit work like making them "trails"). You would need to decide which one is the main one - data, or experiments, or pipelines.
- pipelines are inherently hard to isolate tbh, w/o any data or experiments context / preparation

File: content/docs/start/index.md
Comment (shcheklein) [reply content/docs/start/index.md:81]: No matter if we consolidate or not (again, we had it before in different ways and forms already :) ) - I agree with @daavoo - tbh not clear why we should mention individual subpages. Makes it too lengthy.

File: content/docs/start/index.md
Comment (shcheklein) [reply content/docs/start/index.md]: My 2cs - I would simplify it. When I see Live it forces me to think if there is some special about it? Is it _only_ Live, is there non-live, etc.

File: redirects-list.json
Comment (shcheklein) [inline redirects-list.json:50]: if you are updating 301s I think you need to an extra one  /doc/start/experiments -> to something new. Overall, we should be using 302 in most cases where we iterate on the target and expect target to keep changing.

File: content/docs/user-guide/data-management/discovering-and-accessing-data.md
Comment (shcheklein) [inline content/docs/user-guide/data-management/discovering-and-accessing-data.md:1]: some text on this page becomes out of context I think

File: content/docs/start/index.md
Comment (dberenbaum) [reply content/docs/start/index.md]: Thanks @omesser! This feels worth discussing since it is the first intro to experiments and also can help with our overall messaging.

> The dvclive "values" I think we want to highlight are "live" and "works by instrumenting your code" 😅 maybe we're trying to highlight different things. To explain, the reason I'm trying to say those 2 things is that those are 2 properties/features that competitor tools have and I believe users will understand.

Yeah, I think we have different goals in mind. My thoughts are that since users already likely have familiarity with those properties of other experiment trackers, they are implied and don't explain why to use DVC. If I'm new to DVC, it wouldn't be surprising that I can track experiments using code instrumentation, it would be surprising that DVC records all that in Git.

I can see how mentioning Git could be confusing without going into too much detail here, so this part is not a blocker to me.

> I've seen it refer to as a build tool / system (e.g. "makefile for ML projects") - I think that works well. more accurately a build tool than an orchestrator, but there's a drawback - "makefile" and "build system" might not resonate well with our target audience, but more with hardcore SW engineers.

We already mention it as a build tool in data management. My "orchestrator" suggestion was trying to separate data pipelines (auto build and skip cached stages) and experiment pipelines (queue and execute many variations). Maybe orchestrator isn't right, but I think that `dvc exp run --queue --set-param lr=range(0.01,0.1,0.01) --set-param opt=adam,sgd` goes beyond a build tool.

I also worry that both this paragraph and the reordering of the pages makes the collaboration seem dependent on the pipelines, which I think was why @daavoo had put that ahead of pipelines before. Overall, I think @daavoo has a point that it may be better ultimately to separate pipelines than explain them in both data management and experiment contexts. 

To focus on the main value prop of experiments, I'd be in favor of dropping pipelines from this paragraph for now and shortening to something like:

> Easily track your experiments and their progress by only instrumenting your code, and collaborate on ML experiments like software engineers do for code.

Comment (omesser) [reply content/docs/start/index.md]: Thanks for the explanation Dave!
It's good to hear more opinions here.

This goes into tangent a bit but:

> Maybe orchestrator isn't right, but I think that dvc exp run --queue --set-param lr=range(0.01,0.1,0.01) --set-param opt=adam,sgd goes beyond a build tool.

For the record, I would still prefer "build" tool or "job-runner" here. we're going with build tool for now everywhere else, so it's ok to stick with this imo.
Orchestration of workloads (like jobs, tasks, experiments) beyond that usually - things like allocation of jobs to resources (machines or specific cpu/gpu/mem allocation), auto-healing, moving jobs around to optimize according to different cluster/workload parameters... usually this supports integration with monitoring for the sake of more advanced execution scenarios, etc...  all those are wayyy out of scope. I do think we're dealing with "makefile" type task runner on steroids (because, pipelines).


> To focus on the main value prop of experiments, I'd be in favor of dropping pipelines from this paragraph for now and shortening to something like:
Yep, thanks for "compromising" here. If we go as far as making a decision to isolate pipelines it will expand the scope a lot here. so those short-short descriptions need to be super concise (and remember, when/if we move pipelines out, and I won't block that, this can be adjusted)

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md]: > As a counterpoint, DVC pipelines was the primary value prop for me adopting it as a user.

That's a good point. I'm still on the fence - we "want" to sell this, but I feel pretty strongly about people having different connotation when they see "pipelines". In the context of exp management (usually single stage stupid task managers for competitors) - it's a powerful implementation. In the context of pipeline orchestrators - we don't compete (or, don't compete that well, imo).

Comment (omesser) [reply content/docs/start/index.md]: @daavoo  - nothing to apologize for. a great point, I hope it will drive the next PR here and then we'll attack this problem closer to its "core"

Comment (omesser) [reply content/docs/start/index.md]: For now, I re-ordered as I said, so resolving this thread

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md:81]: @shcheklein  - I think it is coupled with single/non-single page honestly, because:
- now chapters are self contained to a sense, you an drop in any of them. Making only 1 chapter an entry point makes the next ones automatically non-entry points and that would require most content changes, it will basically be 1 piece of content per trail, either single page or some wizard with steps (pages), but it would make it so no one would drop on the non-first chapter, so no point in keeping them all self-contained.
 most people don't notice the next/prev buttons, and navigate by links, so they might not notice and miss those nest pages without proper hooks (press next to read about...)

Comment (omesser) [reply content/docs/start/index.md:81]: Again, if no strong objections, let's leave this to a followup PR please

File: content/docs/start/index.md
Comment (dberenbaum) [reply content/docs/start/index.md]: Makes sense, thanks @omesser! Totally agree about keeping it concise here, so I guess we are agreed to drop the pipelines part of this paragraph and call this resolved?

File: content/docs/user-guide/data-management/discovering-and-accessing-data.md
Comment (omesser) [reply content/docs/user-guide/data-management/discovering-and-accessing-data.md:1]: gave this another read, lgtm. Did some minor copy-editing to the initial paragraph.
To clarify, I don't think referring to the 'get-started' dir of https://github.com/iterative/dataset-registry is problematic.

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md:143]: This was here before in some other form as you can see. I thought about removing it entirely, but I guess having some navigation-helper text in SOME place is helpful, so we have it in 1 place, and what other place is more suitable then in the first page a new user sees.
Maybe it's ok to keep since, like I mentioned in another comment, keep in mind users might not notice the next/prev buttons 😄

File: content/docs/start/experiments/experiment-pipelines.md
Comment (omesser) [reply content/docs/start/experiments/experiment-pipelines.md:2]: I think you're hitting a much larger issue here (design? product?). one that get-started doc cannot fully resolve, and def not in this PR :) why do we have `dvc exp run` and not `dvc pipeline run` ? 😉 
Using the experiment commands and terminology as synonyms to pipelines is indeed super confusing... this is why @daavoo and @dacbd are in favour of consolidating the pipeline contents to its own section. which would make sense for this aspect, but would create another discrepancy where the pipeline content won't be strongly associated with experiment section...
It's a deeper issue indeed.

> Making exps reproducible

This is not a good title for get started I think. it's too "advanced", focused on a feature of a thing and not the thing. The deeper issue here is that we don't have several names for pipelines: data-pipelines, pipelines and experiments

Comment (omesser) [reply content/docs/start/experiments/experiment-pipelines.md:2]: To trace back my mental steps, I initially called this: **experiments as pipelines**
But it didn't look as clear to me as a title / section and page name, so I condensed it a bit 😄

File: redirects-list.json
Comment (omesser) [reply redirects-list.json:50]: This line change is a mistake I forgot to undo! reverted. I'm not doing anything fancy here, no need to add a redirect, `/doc/start/experiments` is the same before and after this PR (I changed to experiment-management and then back because I saw that it is confusing because of a user-guide page with the same name)

File: content/docs/start/experiments/experiment-pipelines.md
Comment (shcheklein) [reply content/docs/start/experiments/experiment-pipelines.md:2]: so, let's do just "pipelines" for now? My point is - let's not invent and reinvent wheels please (like live tracking, etc). It's important to keep it simple. Any adjective immediately raises a lot of questions. It's better to be very explicit and "dumb" vs smart and eloquent, etc. I hope that makes sense.

Again, in this case I would personally go with some actual task that we are solving. Even pipelines are too abstract to my mind for this section.

File: content/docs/start/index.md
Comment (shcheklein) [reply content/docs/start/index.md:143]: I think it's related to the discussion on this page content above. I think the page is way toooooo long. Also We already have a lot of navigation help there. I think two links with some descriptions to the top pages of trails should be enough to serve the "navigational" purpose.

File: content/docs/start/experiments/experiment-pipelines.md
Comment (omesser) [reply content/docs/start/experiments/experiment-pipelines.md]: @dberenbaum  - Got it, I understand the concern. Expanded scope a bit, but I massaged `experiment-pipelines.md` a bit

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md:142]: @shcheklein  - If you feel strongly about this let me and I'll remove

File: content/docs/start/experiments/experiment-pipelines.md
Comment (omesser) [reply content/docs/start/experiments/experiment-pipelines.md:2]: > so, let's do just "pipelines" for now?

The point of this PR is to get some order and logic into those chapters so they make sense in the context of the trails and be meaningful. This chapter deals with using pipelines for experiments and using `dvc exp run` on them. It has to be distinct from the data pipelines chapter under data-management and associated with experiments somehow to be meaningful and clear.
This is not inventing or re-inventing anything here, just removing vagueness and confusion that hit you when you're reading this page today.

I will go with "Experimenting using pipelines" in this case, and will have to rename other chapters to make them a bit more consistent, it will make them less compact.
I don't think it's better but that's the only other way to reconcile our differences here I think. I truely think that naming this "pipelines" with the current structure is counter productive. 

>  It's important to keep it simple. Any adjective immediately raises a lot of questions.

I am a firm believer that clarity are more important than brevity 😄 

> Again, in this case I would personally go with some actual task that we are solving. Even pipelines are too abstract to my mind for this section.

I'm sorry I'm not getting this part I guess. This is not a real life tutorial but get-started, but I think "experimenting using pipelines" is maybe close to what you meant here. If I understand correctly

Comment (shcheklein) [reply content/docs/start/experiments/experiment-pipelines.md:2]: > I am a firm believer that clarity is more important than brevity 😄

yes 100%, imo, and in this case Building Pipelines is more clear vs "Experiments Pipelines". That's more or less the essence of my comment. 

> This is not inventing or re-inventing anything here, just removing vagueness and confusion that hit you when you're reading this page today.

if you are trying to optimize a case when person lands to this page and you want them to see "Experiments" (it's already in URL, it should hopefully clear from the text that this belongs to a trail, but anyways), I would make the _title_ (not the sidebar entry), something like "Experiments: Building Pipelines" (just an example).

My point in all this discussion (similar to the `live ..`) is that any adjective or modifier double the mental effort needed to understand this. E.g. "Simple Pipelines" (are there complex? what's the difference, etc). In this case I read "Experiments" as modifier (not a as context) and it raises questions in my head. 

> It has to be distinct from the data pipelines chapter under data-management and associated with experiments somehow to be meaningful and clear.

It doesn't have to I think, btw. They both could have the same title - and I would be personally fine with that. I think we are bit overcomplicating this. Also, clearly pipelines is a topic on which we don't have a clear decision / discussion yet.

File: content/docs/start/index.md
Comment (shcheklein) [reply content/docs/start/index.md:142]: > Oh, that's a different https://github.com/iterative/dvc.org/pull/4460#discussion_r1162708147 then, let's try to separate from this single sentence please.

they are related to my mind, since this sentence serves a navigational purpose _and_ the whole content on the page does the same I think.

> This content was here in this form before:

Yep, it was more a less educational stub about the expandable section. So, it's a completely different meaning to me now. And in the new (very different form and for a different reason) it feels excessive to me.

> Do you still want this completely removed?

your call, I shared my opinion on this (and on the whole page), it's not a blocker :)

> split it - chapter navigation help should come BEFORE the chapter links

It depends on the page length. If everything fits into one screen ( no scroll) - it doesn't matter much where it stays. I think initially the page was way smaller (before we started introducing trails).

File: content/docs/start/index.md
Comment (shcheklein) [reply content/docs/start/index.md:80]: no strong opinion, I see your point, I would still decouple it (from a single page decision, there was no intention to do it this way initially, I'm not sure it's 100% done this way, etc) and simplify the page. If get started starts with a 2-3 screen page just to navigate it - something is wrong with it fundamentally. 

To clarify, I think there are many ways to keep even all these links, give ppl about the structure _and_ keep it preferably ~1 screen. (e.g. reduce explanations, embed aggressively into paragraphs on the main trails explanation, put a list of links one by one after the trail desc, to cards, etc, etc).

Anyways, to clarify, not a blocker on my end.

File: content/docs/start/experiments/experiment-pipelines.md
Comment (omesser) [reply content/docs/start/experiments/experiment-pipelines.md:2]: I understand, we may not being in agreement about the role of modifiers and adjectives though.
To try and move this forward I pushed the change so it is now "Experimenting using pipelines".
If/when we condense the content further so "experiments" will be a single page, or multi-page but with a specific entry points, and then "pipelines" or "building pipelines" will just be a part of it in a clear context, I would agree with this change, but currently, with chapters being self-sufficient I want the differentiation to be very clear.

> clearly pipelines is a topic on which we don't have a clear decision / discussion yet.

Many discussions from what I've heard ;) not a decision

PTAL at the changes now, tell me if we're good to go

Comment (shcheklein) [reply content/docs/start/experiments/experiment-pipelines.md:2]: it's ready when you feel it's ready :)

> but currently, with chapters being self-sufficient I want the differentiation to be very clear

yep, as a reminder - there are ways to this by making the title (what you on see on the page, not sidebar) very explicit - you have way more space there and it's good for SEO.

File: content/docs/start/index.md
Comment (omesser) [reply content/docs/start/index.md:142]: If it's ok with you then, let's keep it for now.
I will do a followup PR as I mentioned to try and condense more content and maybe unify pages in the trails and this will effect this as well
As per fitting in one screen or not - I think order has to be correct regardless personally. people have different screen sizes, orientation, may read this on mobile / tablet and resize, so we shouldn't count on it too much

Comment (shcheklein) [reply content/docs/start/index.md:142]: sounds good!

>  condense more content and maybe unify pages in the trails and this will effect this as well

My 2cs.

I don't feel it's a top priority tbh. If we decide to do this I would think about unifying and making it a single trails again (which has certain challenges).

Things like - let's remove a bit of admons / expandables and or move content into one page w/o deeply changing it - my personal take - they won't move the needle. 

We went from a single page (Quick Start) -> to 3-4 pages (no experiments, no metrics yet, no import/get) -> 7-8 pages with import/get, metrics -> to 3-4 that combine all that stuff -> to now trails since we want to make experiments a bit more of first-class citizen. In my experience - I didn't see much difference so far in all of those. 

It well might be that now we have more ppl on the website, etc (environment is different). 

Anyways, just saying - 2 trails with multiple pages most likely is the same a single trail and most likely is more or less the same as 2 tails with a single page is. The meaningful change is to make it like MLEM, but in case of DVC I don't think it's possible tbh. They are very different products, with different learning curve, etc, etc.

File: content/docs/start/experiments/experiment-pipelines.md
Comment (omesser) [reply content/docs/start/experiments/experiment-pipelines.md:198]: Ended up expanding scope a bit and editing this content so we don't hang in / explain too much about the basic pipeline building stuff (add stage), but mention the commands and progress

**Comment by omesser** [comment]
@dberenbaum @daavoo @shcheklein  - would love to get an approval to unblock this, or review if you still see something that bothers you in this scope or that regressed from current upstream that I might have missed

Thanks!

**Comment by shcheklein** [comment]
> 👍 I think these changes were warranted as relatively quick fixes to confusion that was identified by both internal feedback and from conversations from users after the recent changes, but agreed we don't need to spend much more time here, since we have spent a lot of iterations already and haven't seen much evidence it has major impact.

yep, some of them were quick fixes, some of them (like trails, or migration from one page to a reproducible project) were pretty large and planned activities

**Comment by dberenbaum** [comment]
Sorry, I meant this PR was warranted because it was mostly about relatively quick fixes. I didn't mean to suggest all the get started work we have done was quick fixes 😄 .

**Pull request merged**
